index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- linking fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <!-- lnking  -->
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="navbar">
+            <p class="logo">my logo</p>
+            <img class="burger" src="graphics/burger-menu.svg" alt="">
+            <!-- <nav>
+                <ul>
+                    <li><a href="">portfolio</a></li>
+                    <li><a href="">about</a></li>
+                    <li><a href="">contact</a></li>
+                </ul>
+            </nav> -->
+        </div>
+        <div class="intro">
+            <div class="flex-container">
+                <h1>
+                    <span class="name">i'm mariane</span>
+                    <br>
+                    <span class="glad">glad</span> to see you!
+                </h1>
```

Comment:
- [ ] Kindly be noticed that the main title (e.g., "I'm Marianne ...") is a `h1` tag and the subtitle (if existing) uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)` so you can but `I'm mariane` in an `h1` tag and `glad to see you!` in an `h2` tag.